deepce.sh

```diff
@@ -1112,6 +1115,92 @@ exploitDockerSock() {
   # TODO: Tidy up command
 }
 
+exploitSysModule(){
+  caps=$(capsh --print)
+  if [[ ! $caps =~ "cap_sys_module" ]];then
+    printError "We don't have the SYS_MODULE capability, which is required for this exploit"
+    exit 1
+  fi
+
+  if [[ -z "$ip" ]];then
```

Comment:
```suggestion
  if [ -z "$ip" ];then
```

Double square brackets do not work in sh (might need to re-test this still works)